src/content/docs/faq.md

```diff
@@ -347,6 +347,25 @@ lines. Here's an example using [`tui-big-text`](https://github.com/joshka/tui-bi
 
 ![[tui-big-text](https://github.com/joshka/tui-big-text/)](https://camo.githubusercontent.com/3a738ce21da3ae67660181538ef27473b86bebca73f42944e8012d52f86e500d/68747470733a2f2f7668732e636861726d2e73682f7668732d3364545474724c6b79553534684e61683232504152392e676966)
 
+## Some characters appear to be missing / look weird
+
+`ratatui`, and TUIs in general, use special drawing characters like [box-drawing characters],
+[braille characters] or even icons. If your font doesn't support such features, it may display a
+white square (□) or a replacement character (�).
+
+[box-drawing characters]: https://en.wikipedia.org/wiki/Box-drawing_character
+[braille characters]: https://en.wikipedia.org/wiki/Braille_Patterns#Block
+
+To fix this, we recommend you use a [nerd fonts], this is generally what works the best. Though note
+that some characters may render a bit differently from font to font.
+
+[nerd fonts]: https://www.nerdfonts.com/
+
+In addition, some terminals like [Alacritty], use a special, separate font to render box drawing
+characters correctly.
+
+[Alacritty]: https://github.com/alacritty/alacritty/blob/master/extra/man/alacritty.5.scd#font
```

Comment:
I'm a bit annoyed that I can't link to the exact place where this is stated, which is a little bit lower: _builtin_box_drawing_

Reply:
You could make text part of the link like this:

```
[Alacritty (See `builtin_box_drawing`]: https://github.com/alacritty/alacritty/blob/master/extra/man/alacritty.5.scd#font
```